templates/18-04/index.html

```diff
@@ -0,0 +1,215 @@
+  </div>
+  <div class="row">
+    <div class="col-9 col-start-large-4">
+      <hr class="is-fixed-width" />
+      <div class="row">
+        <div class="col-3">
+          <h3 class="p-heading--5">Infrastructure packages</h3>
+        </div>
+        <div class="col-6">
+          <p>For Ubuntu 18.04 LTS, where technically feasible, Canonical provides extended security maintenance to all binary packages that reside in the Ubuntu Main Repository.</p>
+          <p>OpenStack Queens packages are security maintained in Ubuntu Pro 18.04 LTS until 2028.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-9 col-start-large-4">
+      <hr class="is-fixed-width" />
+      <div class="row">
+        <div class="col-3">
+          <h3 class="p-heading--5">Application packages</h3>
+        </div>
+        <div class="col-6">
+          <p>Canonical provides application extended security maintenance to the binary packages that reside in Main and Universe. A few commonly-used packages include:</p>
+          <ul class="p-list--divided">
+            <li class="p-list__item">MySQL - 5.7</li>
+            <li class="p-list__item">Python - 2.7</li>
+            <li class="p-list__item">PostgreSQL - 10</li>
+            <li class="p-list__item">Ruby - 2.5</li>
+            <li class="p-list__item">Nodejs - 8.10</li>
+            <li class="p-list__item">PHP - 7.2</li>
+            <li class="p-list__item">ROS - Melodic</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
+  <hr class="is-fixed-width" />
+  <div class="row">
+    <div class="col-6">
+      <h2>What customers say <br class="u-hide--small">about ESM</h2>
+    </div>
+    <div class="col-6">
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">ESM literally saved our lives. It's allowing us to upgrade from 14.04 LTS at our own pace. It's taken the pressure off, and it also means we can tackle the Ubuntu upgrades at the same time as we roll out the new version of our platform.</p>
+        <span class="p-pull-quote__citation">Zivago Lee, Director of DevOps Engineering, Interana</span>
+        <a href="/engage/interana-casestudy">Read the case study&nbsp;&rsaquo;</a>
+      </blockquote>
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">ESM has given us the space to plan what comes next. It's helping us get into a position where we can have a more sustainable infrastructure.</p>
+        <span class="p-pull-quote__citation">Andy Parker, Engineering Manager, TIM</span>
+        <a href="/engage/case-study-acuris">Read the case study&nbsp;&rsaquo;</a>
+      </blockquote>
+      <hr />
+      <p><a href="/security/esm#get-in-touch">Get in touch</a> if you need advice on the best path for your company.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep">
+  <div class="row">
```

Comment:
From the design, looks like there should be an hr above this row to separate this section